src/queries/api.js

```diff
@@ -266,6 +366,112 @@ const api = {
           }),
     };
   },
+  assets() {
```

Comment:
dang, a bunch of new endpoint here. A lot of work, great job. 💪

Reply:
Thanks, so many views on this page! It will be cool to expand this to all asset types if needed.